backend/app/Controllers/Admin/OidcProvidersController.php

```
@@ -0,0 +1,227 @@
+            new OA\Response(
+                response: 200,
+                description: 'Providers retrieved successfully',
+                content: new OA\JsonContent(
+                    properties: [
+                        new OA\Property(property: 'providers', type: 'array', items: new OA\Items(ref: '#/components/schemas/OidcProvider')),
+                    ]
+                )
+            ),
+        ]
+    )]
+    public function index(Request $request): Response
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)